src/backend/parser/parse_expr.c

```diff
@@ -1522,10 +1532,12 @@ transformFuncCall(ParseState *pstate, FuncCall *fn)
        */
 
       if (!schemaname || (strlen(schemaname) == 3 && strncmp(schemaname, "sys", 3) == 0))
-              if (strlen(functionname) == 8 &&
-                          strncmp(functionname, "checksum", 8) == 0 &&
-                          fn->agg_star == true)
-                      targs = ExpandChecksumStar(pstate, fn, fn->location);
+	{
+        if (strlen(functionname) == 8 &&
+            strncmp(functionname, "checksum", 8) == 0 &&
+                fn->agg_star == true)
```

Comment:
nit:
seems like a hot code path so may be worth to optimize.
Check fn->agg_start first before strlen.

Reply:
I didn't change this code, it's a format change for our babel change previously